lib/set-api-origin.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const parseNestedQuery = require('./parse-nested-query');
+
+/*
+  possibilites to handle:
+
+  front-end local, api local
+  front-end local, api remote - prod=true
+  front-end remote, api local - dev=true
+  front-end remote, api remote
+*/
```

Comment:
I like that.  I usually don't like comments that just say what the code does, but this seems useful.